sass/_buttons.scss

```diff
@@ -1,9 +1,3 @@
-.button-container {
-  display: table;
-  margin-left: auto;
-  margin-right: auto;
-}
-
 button,
 .button,
```

Comment:
One oddity about this is that there seems to be two entries for `.button`, with the other being in `_pagination.scss`. Not sure if that's intentional or we can perhaps consolidate/remove the `_pagination.scss` code.

Reply:
tbqh, my porting of CSS to SCSS was an intelligent guess at best and I would not be surprised if there were further optimizations like this here. for the sake of getting this merged, I defer to you (or anyone else) to investigate this, if ever.